.circleci/config.yml

```diff
@@ -77,7 +77,7 @@ jobs:
           name: Start environment
           command: |
             set -euo pipefail
-            docker-compose up -d
+            docker-compose up -d web mockserver
```

Comment:
> Turns out a `watch` container was also started, which would cause unnecessary reload of the API

Cool, makes sense not to have it here, maybe I'd just add a comment above explaining that and linking to this PR but that's up to you :slightly_smiling_face: